### ISSUE TITLE
win_updates - check wsus connectivity

### ISSUE THREAD

**Issue by hoerup** [ansible/ansible#47110]
##### SUMMARY
Have win_updates (or a helper module) check for wsus connectivity before starting the update process
I have added a simple helper task that sortof does the same - but would be nice to have this as an integrated feature.

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
win_updates

##### ADDITIONAL INFORMATION

    - name: Check wsus configuration
      register: wsuscheck
      win_shell: |
        if ( -Not (test-path "HKLM:\SOFTWARE\Policies\Microsoft\Windows\WindowsUpdate") ) {
            Write-Host "Policy path not found"
            exit 1
        }
        $wsusServer = (Get-ItemProperty "HKLM:\SOFTWARE\Policies\Microsoft\Windows\WindowsUpdate").WUServer
        if ($wsusServer -eq $None) {
          Write-Host "Policy not set"
          exit 2
        }
        $response = Invoke-WebRequest -Uri $wsusServer -UseBasicParsing

**Comment by ansibot** [comment]
Hi @hoerup, thank you for submitting this issue!

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
Files identified in the description:
* [lib/ansible/modules/windows/win_updates.ps1](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/windows/win_updates.ps1)

If these files are inaccurate, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @dagwieers @jborean93 @jhawkesworth @nitzmahone
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @if-meaton
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
Files identified in the description:
* [lib/ansible/modules/windows/win_updates.py](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/windows/win_updates.py)

If these files are inaccurate, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @ShachafGoldstein
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by jborean93** [comment]
I'm not sure I see the added benefit here, if WSUS isn't available then `win_updates` will fail anyway. Having the WSUS server unavailable just means it will have to fail at another spot and unless you have some external way of kicking the server in an Ansible playbook there is not much the author can do.
